client/utils/currency/index.js

```diff
@@ -68,6 +68,48 @@ export const getCurrency = ( currencyCode, baseCurrencyCode = null ) => {
 };
 /* eslint-enable valid-jsdoc */
 
+/**
+ * Gets wc-admin currency for the given currency code. Unlike `getCurrency`, this function
+ * will return the currency based only on the currency itself and locale. This means that the
+ * currency object will not be modified based on the store's country settings,
+ * or on another given currency.
+ *
+ * @param {string} currencyCode Currency code
+ * @return {Object} Currency object.
+ */
+export const getCurrencyByLocale = ( currencyCode ) => {
```

Comment:
I created this new helper because I didn't want to risk messing with how `getCurrency` and `formatCurrency` work in other parts of the project. The outcome of this function is the same as in `getCurrency` but it works a little bit differently.

Reply:
I believe it was a good trade-off between code repetition and reduced risk of regressions.